### DISCUSSION TITLE
CUBE CSS: uma nova metodologia CSS

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1845] [Dicas]
Há um tempinho atrás, tinha lido algo no CSS-Tricks sobre uma tal **CUBE CSS**, mas não dei muita bola.

Recentemente, vi alguns tweets e, ontem, o criador da coisa, [Andy Bell](https://twitter.com/hankchizljaw), anunciou que uma [documentação oficial](https://piccalil.li/cube-css/) tinha sido publicada.

Mais do que depressa resolvi ver melhor e, no final, achei bem interessante surgir uma nova metodologia CSS. Ainda estou estudando, de primeira, achei alguns pontos bem esquisitinhos, mas, de qualquer maneira, resolvi contribuir com a coisa.

Aqui está a versão brasileira não autorizada do CUBE CSS.  =P

https://desenvolvimentoparaweb.com/css/cube-css/

Alguém já tinha ouvido falar e/ou já brincou um pouquinho para tirar opiniões mais embasadas?

**Comment by nazarepiedady** [comment]
Muito bem!
Você faz um excelente trabalho, obrigado!
